src/main/java/com/koczy/kurek/mizera/thesisbrowser/entity/Author.java

```diff
@@ -31,14 +31,14 @@ public Author() {
     }
 
     public Author(String name, Date birthDate, String academicTitle, String university) {
-        this.name = name;
+        this.name = WordUtils.capitalize(name);
```

Comment:
Po co wszystko dużymi literami ?

Reply:
Ta funkcja zmienia pierwsze litery słów na duże. Ale chyba to nic nie dało bo i tak czasami do bazy zapisuje dziwnie. @Thomasz111  wiesz cos o tym?